test/test_url.rb

```diff
@@ -80,5 +80,12 @@ class TestURL < JekyllUnitTest
         ).to_s
       end
     end
+
+    should "ignore colons in placeholders" do
```

Comment:
I feel this hasn't been named correctly. Are colons being ignored or simply no longer considered a delimiter?